04_API/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Use the official lightweight Python image.
```

Comment:
Sacado de aquí: https://cloud.google.com/run/docs/quickstarts/build-and-deploy/deploy-python-service